tests/Console/Command/StageCommandTest.php

```diff
@@ -30,22 +47,88 @@ public function testBasicConfiguration(): void
         $definition = $command->getDefinition();
         $arguments = $definition->getArguments();
         $options = $definition->getOptions();
+        $composerCommandArgument = $definition->getArgument('composer-command');
 
         self::assertSame('stage', $command->getName(), 'Set correct name.');
         self::assertSame([], $command->getAliases(), 'Set correct aliases.');
         self::assertNotEmpty($command->getDescription(), 'Set a description.');
-        self::assertSame([], array_keys($arguments), 'Set correct arguments.');
+        self::assertSame(['composer-command'], array_keys($arguments), 'Set correct arguments.');
         self::assertSame([], array_keys($options), 'Set correct options.');
+        self::assertNotEmpty($command->getUsages(), 'Set usages.');
+        self::assertNotEmpty($command->getHelp(), 'Set help.');
+
+        self::assertTrue($composerCommandArgument->isRequired(), 'Required Composer command option.');
+        self::assertTrue($composerCommandArgument->isArray(), 'Required Composer command option.');
```

Comment:
Copy/paste error on this second assertion.
```suggestion
        self::assertTrue($composerCommandArgument->isArray(), 'Set Composer command option to array.');
```